.goreleaser.yml

```diff
@@ -2,8 +2,9 @@ before:
   hooks:
     - go mod download
 builds:
-  - main: ./cmd/attestation.go
+  - main: ./main.go
     binary: attestation
+    ldflags: -X github.com/liatrio/gh-trusted-builds-attestations/build.Version={{.Version}}
```

Comment:
Should this use`.Tag` instead? `.Version` will remove the `v` prefix. I could go either way

https://goreleaser.com/customization/templates/?h=.version#common-fields